### PR TITLE
Update test cases for features added in v1.3

### PR DESCRIPTION
**Pull request by maj0-0**
Haven't added test case for testscommand, still working on it.
If i missed anything lmk!

### CODE REVIEW

File: src/main/java/seedu/address/model/group/tasks/Deadline.java [hunk 1]
```diff
@@ -38,39 +36,39 @@ public String getDeadline() {
...
+//    /**
+//     * Generates a string representation of the `Deadline` task for saving to a file.
+//     *
+//     * @return A formatted string representing the `Deadline` task for file storage.
+//     */
+//    @Override
+//    public String generateStr() {
+//        return "D | " + this.getStatus()
+//            + " | " + this.getTask() + " | " + byStr + " | " + this.getModule();
+//    }
```
Comment (Bearypop) [inline src/main/java/seedu/address/model/group/tasks/Deadline.java:73]: These parts are not needed?

**Review by Bearypop — COMMENTED** [review]
Looks good

File: src/main/java/seedu/address/model/group/tasks/Deadline.java (hunk 1 above)
Comment (maj0-0) [reply src/main/java/seedu/address/model/group/tasks/Deadline.java:73]: we dont use anywhere but jus commented out to be safe for now

Comment (wnchan) [reply src/main/java/seedu/address/model/group/tasks/Deadline.java:73]: i think can delete it if we don't use.

**Review by wnchan — COMMENTED** [review]
LGTM!

**Review by neyapraveen — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/model/group/tasks/Deadline.java (hunk 1 above)
Comment (maj0-0) [reply src/main/java/seedu/address/model/group/tasks/Deadline.java]: thanks! done!

**Pull request closed without merging**